packages/teleport/src/Main/Main.story.tsx

```diff
@@ -64,12 +64,12 @@ function useMainStory() {
     ctx.auditService.fetchEvents = () =>
       Promise.resolve({ events, startKey: '' });
     ctx.clusterService.fetchClusters = () => Promise.resolve(clusters);
-    ctx.nodeService.fetchNodes = () => Promise.resolve(nodes);
+    ctx.nodeService.fetchNodes = () => Promise.resolve({ nodes });
     ctx.sshService.fetchSessions = () => Promise.resolve(sessions);
```

Comment:
Why were `clusters` and `sessions` left out of this structure change?

Reply:
they are not affected by the change that is coming (yet)